src/components/new-request/search.vue

```diff
@@ -578,49 +595,47 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin) {
   }
 
   /** Called when Request Action menu item is changed. */
-  onRequestActionChange (request: RequestActionsI): void {
+  async onRequestActionChange (request: RequestActionsI): Promise<void> {
```

Comment:
Does it have to return `Promise<void>`? Wouldn't be sufficient to be just `void` Sev?

Reply:
Nope, because it's async, so technically it returns a promise.

Remember that await is equivalent to ".then().catch()", if that helps you figure out promises.